Resources/Prototypes/_NF/Catalog/VendingMachines/Inventories/cart.yml

```
@@ -3,6 +3,8 @@
   startingInventory:
     ContractorPDA: 4294967295 #infinite
     ClearNFPDA: 4294967295 #infinite
+    MercenaryPDA: 4294967295 #infinite
+    PilotPDA: 4294967295 #infinite
```

```suggestion
    PilotPDA: 4294967295 #infinite
    MercenaryPDA: 4294967295 #infinite
    ClearNFPDA: 4294967295 #infinite
```
Better order, keeps the job-related ones together.

---

It should be:

ClearNFPDA: 4294967295 #infinite
PilotPDA: 4294967295 #infinite
MercenaryPDA: 4294967295 #infinite

no?